nextcord/abc.py

```diff
@@ -1366,45 +1372,50 @@ async def send(
         state = self._state
         content = str(content) if content is not None else None
 
+        embed_data: Optional[EmbedData] = None
```

Comment:
I feel that _payload sounds better than _data for these variable names. Feel free to disregard if you don't agree.

```suggestion
        embed_payload: Optional[EmbedData] = None
```